libfreefare/ntag21x.c

```diff
@@ -113,14 +117,14 @@ _ntag21x_tag_new(nfc_device *device, nfc_target target)
 	tag->device = device;
 	tag->info = target;
 	tag->active = 0;
-	NTAG_21x(tag)->subtype = NTAG_213; // Set tag subtype
-	NTAG_21x(tag)->vendor_id = 0x04;
-	NTAG_21x(tag)->product_type = 0x04;
-	NTAG_21x(tag)->product_subtype = 0x02;
-	NTAG_21x(tag)->major_product_version = 0x01;
+	NTAG_21x(tag)->subtype = NTAG_UNKNOWN; // Set tag subtype to unknown
```

Comment:
The original comment was useless (it just rephrase the actual code), can you remove it instead of updating it?

Reply:
I will :+1: